expected-warnings.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
Since the functionality of the "ignore warnings" is to ignore all warnings, not just duplicate licenses (even if that is the only warnings being generated currently), suggest we change this from a JSON array to a JSON object with 2 properties:

- `duplicates`
- `additionalWarnings`

both would be arrays.

e.g.:

```
{
   "duplicates" : [ ["AGPL ...], ...],
   "additionalWarnings": []
}
```